lib/view_component/base.rb

```diff
@@ -94,6 +96,10 @@ def render?
       true
     end
 
+    def render_content(&block)
+      @block_from_render_content = block
```

Comment:
```suggestion
    def content(&block)
      if block_given?
        @block_from_render_content = block
      else
        @_content ||= block_from_render_content.call 
      end
```

Thoughts on renaming this to `content` and copying the slots API so this is both a getter and a setter?

Reply:
I like it! Adds consistency with the slots API